capabilities.json

```diff
@@ -132,15 +132,28 @@
         "label": {
             "displayName": "Center Label",
             "properties": {
-                "fill": {
-                    "displayName": "Fill",
+                "show": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "color": {
```

Comment:
It's a breaking change. Name of the property has been change. In other words, the existing reports wouldn't be able to get color that was applied before. Please use the previous name instead to avoid any breaking changes.